### PR TITLE
Reactions model

### PR DESCRIPTION
**Pull request by hellen-22**
# Description

Created Reaction model and an abstract model for the created_at and updated_at fields

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sangkips — APPROVED** [review]

File: blog/models.py [hunk 1]
```diff
@@ -6,11 +6,17 @@
...
+class AbstractBaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
 # Category
-class Category(models.Model):
+class Category(AbstractBaseModel):
```
Comment (JimmyTron) [inline blog/models.py:17]: why this?

File: blog/models.py [hunk 2]
```diff
@@ -23,14 +29,12 @@ class Meta:
     (1,"Published")
 )
 """stori is swahili for story. was thinking of using the slang version 'risto'/'riba' in there.. """
-class Stori(models.Model):
+class Stori(AbstractBaseModel):
```
Comment (JimmyTron) [inline blog/models.py:32]: ??

File: blog/models.py [hunk 3]
```diff
@@ -59,4 +63,16 @@ class Meta:
...
+)
+
+class Reaction(AbstractBaseModel):
+    reaction_type = models.CharField(max_length=100, choices=REACTION_TYPE_CHOICES)
+    user = models.ForeignKey(Account, on_delete=models.CASCADE)
+    post = models.ForeignKey(Stori, on_delete=models.CASCADE)
+    comment = models.ForeignKey(Comment, on_delete=models.CASCADE)
 
+    def __str__(self):
+        return self.reaction_type
```
Comment (JimmyTron) [inline blog/models.py:78]: clean

**Review by JimmyTron — APPROVED** [review]
why the use of  ```(AbstractBaseModel)``` though??

**Review by JimmyTron — APPROVED** [review]

**Review by JimmyTron — DISMISSED** [review]

**Pull request merged**